build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eu
+
+test -d binjgb/out || ./build-binjgb.sh
```

Comment:
I'm not sure if this is the behaviour we'd really like to go for? I would expect running the build script to rebuild everything anwyay, not skip it if the files are already there. In this way I'd have to delete the wasm output folder to trigger a re-build?

Reply:
Yes, I agree, _but_ each of our build scripts now apply a patch, which the second time around fails. I assumed that most of the work in this repo happens inside the web frontend, not on these libraries, so optimized for a single build.

I agree that this needs to be smarter, but I was punting that work for a later PR. I can do it in this PR if you prefer!